Source/Diagnostics/Diagnostics.H

```diff
@@ -155,6 +155,10 @@ protected:
     std::vector< int > m_pfield_species_index;
     /** Map connecting particle field variable names to the corresponding parser strings */
     std::map< std::string, std::string > m_pfield_strings = {};
+    /** Whether to use a filter function on particles before calculating particle averages */
+    std::map< std::string, bool> m_pfield_dofilter = {};
```

Comment:
Just realized that we should include `<map>` in this file. Can you add this pls?

Reply:
Good catch. Done.